Feature/refactoring

vzichv opened the pull request.

x-bit-x committed:
Update adaptation for more screens

wotori commented on src/components/NFText/NFText.tsx:
```diff
@@ -0,0 +1,108 @@
+import styles from "./NFText.module.css";
+
+import Image from "next/image";
```
the same comment as in #12 PR

x-bit-x committed:
Change style system

Delete styles.css.
Set styles.module.css how to global styles.
Delete extra lines in js/css + optimization.

x-bit-x committed:
Delete styles.css

wotori committed:
remove page that use Link module for sucecciful build

wotori reviewed (approved):
thanks for refactor and improvements. Please check if you can build the project before submit it tu PR or mark It as WIP in title. We can't merge pr to main if It cant be builded.

x-bit-x committed:
Merge branch 'main' of github.com:wotori-studio/NFText into feature/refactoring

x-bit-x committed:
Partial porting to typescript and code optimization
